### ISSUE TITLE
i18n support

### ISSUE THREAD

**Issue by wengxt** [LibreService/rppi#6]
right now, the name field in json is hard coded in Chinese and there’s no relevant mechanism for i18n’ed name.

**Comment by eagleoflqj** [comment]
Yes, this needs improvement. Do you have any recommendation? e.g. embedding in the same file or using separate json files for translation only? It's not a big deal to break existing structure since only My RIME is using it for now.
I think some names are hard or meaningless to translate.
